mod.cpp

```diff
@@ -1,14 +1,14 @@
-name = "Gruppe Adler GRAD Pace Count Beads";
-author = "Salbei";
+name = "Gruppe Adler Pace Count Beads";
+author = "Ainer, Salbei";
```

Comment:
Ainer ist kein Author!
```suggestion
author = "Salbei";
```